src/utilities/compareSCO.js

```diff
@@ -0,0 +1,56 @@
+import _ from 'lodash';
+
+const defaultState = {
```

Comment:
can we import the default state from somewhere else.  It would become very easy for this to become out of sync.  The default state is exported from form.js

Reply:
Then we should also import the defaultState inside queryStringToSearchCriteria.js - defaultState in form.js and in that util differs.

Should I update the form.js 's defaultState's values with those in util file and then import it in both queryStringToSearchCriteria.js and compareSCO.js ? @arcepaul